### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by gnerkus**
#### Summary
Closes #536 
Closes #537 
Closes #538 
Closes #539  
Closes #540  
Closes #541

### CODE REVIEW

**Comment by montezume** [comment]
Can you update rollup as well?

File: packages/l10n/package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
   },
   "devDependencies": {
     "chalk": "2.4.2",
-    "cldr": "5.0.1",
+    "cldr": "5.2.0",
```
Comment (emmenko) [inline packages/l10n/package.json]: Can you run the script in the `l10n` package to generate the intl data? Just to make sure that we have up-to-date data.

PS: in #511 I manually used the latest data already, so this version should not change anything in theory.

Comment (gnerkus) [reply packages/l10n/package.json]: Noted

File: packages/l10n/data/time-zones/zh-CN.json [hunk 2]
```diff
@@ -144,11 +144,6 @@
     "abbr": "AKDT",
```
[the remark this answers is not in the document]
Comment (tdeekens) [reply packages/l10n/data/time-zones/zh-CN.json]: /cc @emmenko as I am not sure he notices otherwise.

[the remark this answers is not in the document]
Comment (gnerkus) [reply packages/l10n/data/time-zones/zh-CN.json]: @emmenko

[the remark this answers is not in the document]
Comment (emmenko) [reply packages/l10n/data/time-zones/zh-CN.json]: Hmm then maybe revert the version change, I’ll take a look at it separately

[the remark this answers is not in the document]
Comment (gnerkus) [reply packages/l10n/data/time-zones/zh-CN.json]: The error occurred after a reverted the version change. @emmenko

[the remark this answers is not in the document]
Comment (montezume) [reply packages/l10n/data/time-zones/zh-CN.json]: Yeah I'm seeing it on master as well...

[the remark this answers is not in the document]
Comment (montezume) [reply packages/l10n/data/time-zones/zh-CN.json]: Nevermind, I didn't read the docs 🤦‍♂️ 
```
First, download the data (`core.zip`) for a specific version from the [downloads page](http://cldr.unicode.org/index/downloads).

Then extract the data and copy the `core` folder to this package, and rename it to e.g. `cldr-v35`. Then point the `cldr` library to the folder location.

Run the script, which uses the new data.
```

File: packages/l10n/scripts/generate-l10n-data.js [hunk 3]
```diff
@@ -2,7 +2,7 @@
 const fs = require('fs');
 const path = require('path');
 const shell = require('shelljs');
-const cldr = require('cldr').load(path.join(__dirname, '../cldr-v35'));
```
Comment (emmenko) [inline packages/l10n/scripts/generate-l10n-data.js]: This was the problem

**Pull request merged**